api/src/main/java/io/opencensus/stats/ViewData.java

```diff
@@ -232,9 +232,9 @@ private static String createErrorMessageForWindow(
       View.AggregationWindow window, AggregationWindowData windowData) {
     return "AggregationWindow and AggregationWindowData types mismatch. "
         + "AggregationWindow: "
-        + window
+        + window.getClass().getSimpleName()
         + " AggregationWindowData: "
-        + windowData;
+        + windowData.getClass().getSimpleName();
```

Comment:
May be create another checkArgument() which takes generic Object obj and calls obj.getClass().getSimpleName() only if validation fails.

Reply:
Done, though this might be an overkill since getClass().getSimpleName() should only introduce very little overhead.